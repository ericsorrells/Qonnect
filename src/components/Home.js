// ========================================================================================
import React from 'react';
// ========================================================================================

const Home = () => {
  return(
    <div className="home-container">
      <div className="home-image">
      </div>
    </div>
  )
}

export default Home;