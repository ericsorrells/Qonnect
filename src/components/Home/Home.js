// ========================================================================================
import React from 'react';
// ========================================================================================
import NavBar from '../NavBar/NavBar';
// ========================================================================================

const Home = () => {
  return(
    <div>
      <div className="home">
        Home Page
      </div>
    <NavBar />
    </div>
  )
}

export default Home;