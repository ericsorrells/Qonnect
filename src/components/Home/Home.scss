@import '../../styles/variables';

.home {
  background-color: $bd;
}