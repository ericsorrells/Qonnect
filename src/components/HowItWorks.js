// ========================================================================================
import React from 'react';
// ========================================================================================
import HowItWorksItem from './HowItWorksItem';
// ========================================================================================

const HowItWorks = () => {
  return (
    <div>
      How It Works Component!!
      <HowItWorksItem />
    </div>
  )
}

export default HowItWorks;