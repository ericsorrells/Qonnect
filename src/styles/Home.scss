.home-container {
  height: 500px;
}

// .home-image {
//   background: url('/images/home_image.jpg');   // this is to the 'public' folder
//   height: 100%;
//   width: auto;
//   background-size: cover; 
// }