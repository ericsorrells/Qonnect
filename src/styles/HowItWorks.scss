.how-it-works__container {
  background-image: linear-gradient(to top, $main-color-light, $main-color);
  background-color: #534292;
  padding-top: 50px;
  padding-bottom: 50px;
}

.how-it-works__inner-container {
  display: flex;
  justify-content: space-around;
}