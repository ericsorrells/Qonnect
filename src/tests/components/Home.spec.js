// ========================================================================================
import React from 'react';
import { shallow } from 'enzyme';
import 'jest-enzyme';
// ========================================================================================
import Home from '../../components/Home';
import HomeBillboard from '../../components/HomeBillBoard';
import HowItWorks    from '../../components/HowItWorks';
// ========================================================================================

describe('<Home />', () => {
  const component = shallow(<Home />);

  it('renders correct', () => {
    expect(component).toMatchSnapshot();
  });

  it('renders child components', () => {
    expect(component.containsAllMatchingElements([
      <HomeBillboard/>,
      <HowItWorks />
    ])).toBeTruthy()
  });
});