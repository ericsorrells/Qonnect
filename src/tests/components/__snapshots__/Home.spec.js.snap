// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders correct 1`] = `
<div>
  <HomeBillBoard />
  <HowItWorks />
</div>
`;
